### ISSUE TITLE
Jerky animation, no way to track swipes (Problems migrating to version 1.0.0 from 0.6.0)

### ISSUE THREAD

**Issue by merelj** [letsar/flutter_slidable#225]
Hi there, and thanks a lot for such a great library 👍 

Found several problems after migration:

- there is a visible jerk in animation when slidable is swiped quickly: animation first stops about at about halfway and then continues only after user removes their finger

- there is no way to track left/right swipe for analytics: onNotification does not provide any info on what kind of swipe it is and if it was successfull (openThreshold was reached); I more or less managed to track this in 0.6.0, but in 1.0.0 there is just no way

I know that 1.0.0 is a pre-release for now and you might be working on those problems already, but hope it helps 🙇

**Comment by merelj** [comment]
example of the 1st problem:
![Screen Recording 2021-07-16 at 13 00 45](https://user-images.githubusercontent.com/3993495/125931361-ca54eea4-8e3c-4d87-9a52-a44636024d4f.gif)

**Comment by merelj** [comment]
simplified example:

```
  Widget build(BuildContext context) {
    final screenWidth = MediaQuery.of(context).size.width;
    final extentRatio = actionSideLength / screenWidth;
    return Slidable(
      groupTag: 'external-call-slidable',
      child: Container(child: Text('ya'), height: actionSideLength, width: double.infinity,),
      startActionPane: ActionPane(
        motion: const ScrollMotion(),
        extentRatio: extentRatio,
        children: [
          Container(height: actionSideLength, width: actionSideLength, color: Colors.lime)
        ],
      ),
    );
  }
```

**Comment by letsar** [comment]
Hi @merelj, thanks I've been able to fix it in the 1.0.0-dev.5 version.
Concerning a way to track left/right swipe, you can do it.
If you get the `SlidableController` from a `Slidable's` child, you'll be able to listen to `actionPaneType` which is a `ValueNotifier`. With that in mind, you can then do whatever you want with this value.
